src/rules/common/wip.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable prefer-object-spread */
+//
+// Assigns or removes a "wip" label
+//
+
+import { prMergeableState, prTitle, gitHubApi } from '../helpers';
+
+const defaultLabel = 'wip';
+
+const isWipRegEx = /\b(wip)\b/i;
+
+export default async (label = defaultLabel) => {
+  const mergeableState = prMergeableState;
+  // eslint-disable-next-line no-console
+  console.log('label', label);
+  // eslint-disable-next-line no-console
+  console.log('mergeableState', mergeableState);
```

Comment:
<!--
  0 failure: 
  1 warning:  Seems like a cons...
  
  
  DangerID: danger-id-Danger;
  File: src/rules/common/wip.js;
  Line: 17;
-->

- :warning: Seems like a console command is being used.